src/drivers/intel/cavs/alh.c

```diff
@@ -41,7 +41,7 @@ static int alh_get_hw_params(struct dai *dai,
 {
 	/* 0 means variable */
 	params->rate = 0;
```

Comment:
The rate should be set also. Components SRC and ASRC need it.

This similar hack is also needed for HDA.